Scrypted/livestats.blade.php

```diff
@@ -0,0 +1,10 @@
+<ul class="livestats">
```

Comment:
I don't think scrypted has a queue size and speed? Again, this looks like a copy-paste and unimplemented.